### ISSUE TITLE
Column project_id missing from View:public.po_product_orders

### ISSUE THREAD

**Issue by steveblamey** [uzerpllp/uzerp#88]
Exception logged:

```
#!sql

DataObjectCollection('.POrder.') load failed: SELECT id,order_number,supplier,order_date,due_date,status,type,currency,net_value,base_net_value,raised_by_person,description,project,project_id,plmaster_id,currency_id,created,lastupdated,alteredby,usercompanyid FROM po_product_orders WHERE ((productline_header_id = '4923')) ORDER BY order_date DESC, order_number DESC, id ASC LIMIT 50 ERROR: column "project_id" does not exist LINE 1: ...se_net_value,raised_by_person,description,project,project_id...
```

**Comment by steveblamey** [comment]
Fixes #88 Column project_id missing from View:public.po_product_orders


→ <<cset a93e5e710777>>

**Comment by steveblamey** [comment]
Tune error reporting and sentry logging.
fixes #88
fixes #89
Add error feedback for user
Add uzERP logos to default theme


→ <<cset 6fb49dad1f29>>
